src/test/discourse-api/discourse-context.ts

```diff
@@ -1,5 +1,7 @@
-import rp from 'request-promise';
```

Comment:
I think we probably don't need this file any more. This was just used in the tests, right? We should simplify those if possible. Probably not right now, but you should file an issue so we remember to simplify the tests.